test/incr/utils.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+var sinon = require('sinon');
+var _ = require('underscore');
+
+exports.getTestData = function() {
+	return {
+		key: 'testKey',
+		DateResult: {
+			a: _.random(1, 10)
```

Comment:
In this case DateResult should be object, because it will extend this value:
```
var dateMock = sinon.stub().callsFake(function() {
		_(this).extend(testData.DateResult);
	});
```
`a` is just a random field

Reply:
got it, maybe call it randomKey, someKey or so? just don't like single character variables =)

Reply:
Yep, someKey is ok

Reply:
Renamed